code/game/objects/items/grenades/grenade.dm

```diff
@@ -86,16 +86,16 @@
 		if(!botch_check(user)) // if they botch the prime, it'll be handled in botch_check
 			arm_grenade(user)
 
-/obj/item/grenade/proc/log_grenade(mob/user, turf/T)
+/obj/item/grenade/proc/log_grenade(mob/user, turf/Turf)
 	log_bomber(user, "has primed a", src, "for detonation")
 
 /**
  * arm_grenade (formerly preprime) refers to when a grenade with a standard time fuze is activated, making it go beepbeepbeep and then detonate a few seconds later.
  * Grenades with other triggers like remote igniters probably skip this step and go straight to [/obj/item/grenade/proc/detonate]
  */
 /obj/item/grenade/proc/arm_grenade(mob/user, delayoverride, msg = TRUE, volume = 60)
-	var/turf/T = get_turf(src)
-	log_grenade(user, T) //Inbuilt admin procs already handle null users
+	var/turf/Turf = get_turf(src)
+	log_grenade(user, Turf) //Inbuilt admin procs already handle null users
```

Comment:
If it's only used here, just remove the variable entirely.

Reply:
wait, so just remove both lines?

Reply:
`log_grenade(user, get_turf(src))`